### ISSUE TITLE
front: train edition: if the path is invalid but debug mode is available, the rest of the tabs think it's valid

### ISSUE THREAD

**Issue by Castavo** [OpenRailAssociation/osrd#8832]
### What happened?

When a train has an invalid path and the debug mode is available, the tab isn't shown in red and the schedule table is present.

In the video, at first, an example of the desired behavior, then the actual behavior

https://github.com/user-attachments/assets/a15e6e7b-da3d-4420-9c41-2b18e5960954

### What did you expect to happen?

I expect the form to behave the same way whether the PF is fully broken or whether the debug mode is available:
* pathfinding tab surrounded by a red border
* schedule table not available

### How can we reproduce it (as minimally and precisely as possible)?

1. Create a train schedule on a path where for example the electrification is incompatible
2. Witness

### On which environments the bug occurs?

Dev (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

6edea6d

**Comment by clarani** [comment]
Still here

**Comment by SharglutDev** [comment]
@RomainValls are you still working on it ?

**Comment by RomainValls** [comment]
Going back to it !

**Comment by RomainValls** [comment]
@SharglutDev @Castavo The behavior changed slightly in the app. Now, we don't have to refresh the page to see that the trainSchedule is invalidated. In the case of the video, tried to reproduce, and got "incompatible constraints" in the timetable card, as soon as I updated the trainSchedule. 
Do we need to also have the itinerary tab in red, so the user knows the pathfinding isn't valid before updating it ?

**Comment by SharglutDev** [comment]
As the UI will change later this year, I don't think it's worth to devote to much time for this kind of modifications.

**Comment by Castavo** [comment]
I agree that too much time spent would be a loss, but here @SharglutDev would it really be that big ? 

While reproducing the bug, I can the see that the PF is re-run, as the itinerary flashes for a second. Wouldn't it be a matter of updating some variable after PF is re-run ?


Also, complementary bug: the schedule table is present, even though the PF is failing in that case

**Comment by SharglutDev** [comment]
Fine by me. Is it still a major bug for you ?

**Comment by Castavo** [comment]
I don't know, it's between minor and major... It's a very weird behavior that you don't instantly know the PF is now impossible and the schedule table is still present on an invalid path.

If you think it's minor you can downgrade it.

It's still a bug directly reported by one of our users tho.

**Comment by maelysLeratRosso** [comment]
Cannot reproduce. @Castavo do you still reproduce ?

**Comment by Castavo** [comment]
It's still present for me

https://github.com/user-attachments/assets/8f0abbb4-0f72-4a04-86a9-d5e386be94c6

However, the behavior is actually the same than if I created the train broken from the start: the PF is broken but the tab isn't red and the schedule table is present. 

I at least need to update the title and description but maybe even this is the desired behavior @maelysLeratRosso ?

**Comment by Castavo** [comment]
I have heavily updated the bug title and description, in agreement with @maelysLeratRosso

**Comment by maelysLeratRosso** [comment]
I don't think the title is updated ?
